packages/web/src/features/printing-business/component/DesiredPickUpTimeSelection/_atomic/TimeSlotList.tsx

```diff
@@ -5,12 +5,13 @@ import { getHours, getMinutes, setHours, setMinutes } from "date-fns";
 import { executiveWorkingHourStart } from "@sparcs-clubs/web/constants/printingBusiness";
 
 interface TimeSlotListProps {
+  calendarSize: "sm" | "md" | "lg";
   value: Date;
   onDatesChange: (date: Date) => void;
 }
 
-const TimeSlotListInner = styled.div`
-  width: 300px;
+const TimeSlotListInner = styled.div<{ calendarSize: "sm" | "md" | "lg" }>`
+  width: ${({ calendarSize }) => (calendarSize === "md" ? "450px" : "300px")};
```

Comment:
저번에 말한 "md" 또는 "sm"일 때는 width 100%로 하는 것
```suggestion
  width: ${({ calendarSize }) => (calendarSize === "lg" ? "300px" : "100%")};
```

Reply:
수정했습니다!